bgpd/bgp_route.c

```diff
@@ -8107,8 +8108,10 @@ void bgp_redistribute_add(struct bgp *bgp, struct prefix *p,
 			attr.mp_nexthop_len = BGP_ATTR_NHLEN_IPV6_GLOBAL;
 			break;
 		}
```

Comment:
This brace is mis-indented - and that's making the remainder of this 'case' clause also be mis-indented?

Reply:
Not able to understand how is the brace mis-indented. 
Looks to be the same here: https://github.com/FRRouting/frr/blob/master/bgpd/bgp_route.c#L8100
Please let me know so that it can be fixed.

Reply:
you're quite right - the 'view file' display looks fine, but the indentation looks wrong here in the 'diff' view - sorry about that!